docker-compose-mainnet.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.7'
 
 x-defaults: &defaults
-  image: ghcr.io/beamer-bridge/beamer-agent:v1.0.3
+  image: ghcr.io/beamer-bridge/beamer:v1.0.2-31-gee4fef6
```

Comment:
shouldn't this be v2.0.0

Reply:
Think you are not checking the newest commit